lib/orator/index.js

```diff
@@ -13,11 +13,25 @@ class Orator {
   //   // analytics = false,
   //   // analyticsFilePath = './commands.log.json'
   // } = {}
-  constructor (defaultPrefix) {
+  constructor (defaultPrefix, oratorOptions) {
+    if (!oratorOptions) {
+      oratorOptions = {
+        embed: null
```

Comment:
we should provide a default object here instead of null, we can use the embed from the prefs command as a base